CONTRIBUTING.md

```diff
@@ -381,43 +371,41 @@ You can also refer to Criterion docs for additional options and details.
 ### Commits
 
 It is a recommended best practice to keep your changes as logically grouped as
-possible within individual commits. There is no limit to the number of commits
-any single Pull Request may have, and many contributors find it easier to review
-changes that are split across multiple commits.
+possible within individual commits. There is no limit to the number of commits any single Pull
+Request may have, and many contributors find it easier to review changes that are split across
+multiple commits.
 
-That said, if you have a number of commits that are "checkpoints" and don't
-represent a single logical change, please squash those together.
+That said, if you have a number of commits that are "checkpoints" and don't represent a single
+logical change, please squash those together.
 
-Note that multiple commits often get squashed when they are landed (see the
-notes about [commit squashing](#commit-squashing)).
+Note that multiple commits often get squashed when they are landed (see the notes about [commit
+squashing](#commit-squashing)).
 
 #### Commit message guidelines
 
 A good commit message should describe what changed and why.
 
 1. The first line should:
+   - contain a short description of the change (preferably 50 characters or less, and no more than
```

Comment:
Using `-` for lists helps avoid the confusion between lists and italic / bold text